src/table/components/virtualized-table/hooks/__tests__/use-scroll-hander.test.ts

```diff
@@ -0,0 +1,71 @@
+import { renderHook } from '@testing-library/react';
```

Comment:
the file name should be use-scroll-handler.spec.ts right?

Reply:
Indeed. Will fix 👍